website/members/models/profile.py

```diff
@@ -259,36 +259,36 @@ def language(self):
     )
 
     def display_name(self):
+        # pylint: disable=too-many-return-statements
         pref = self.display_name_preference
         if pref == "nickname" and self.nickname is not None:
             return f"'{self.nickname}'"
-        elif pref == "firstname":
+        if pref == "firstname":
             return self.user.first_name
-        elif pref == "initials":
+        if pref == "initials":
             if self.initials:
                 return "{} {}".format(self.initials, self.user.last_name)
             return self.user.last_name
-        elif pref == "fullnick" and self.nickname is not None:
+        if pref == "fullnick" and self.nickname is not None:
             return "{} '{}' {}".format(
                 self.user.first_name, self.nickname, self.user.last_name
             )
-        elif pref == "nicklast" and self.nickname is not None:
+        if pref == "nicklast" and self.nickname is not None:
             return "'{}' {}".format(self.nickname, self.user.last_name)
-        else:
-            return self.user.get_full_name() or self.user.username
+        return self.user.get_full_name() or self.user.username
+        # pylint: disable=too-many-return-statements
```

Comment:
It seems this line can be removed? Doesnt the line at the top disable it for just this function?